recipes/shapeit/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: shapeit
+  version: '2.r837'
+
+source:
+  fn: shapeit.v2.r837.GLIBCv2.12.Linux.static.tgz                                                                  # [linux]
+  url: https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.v2.r837.GLIBCv2.12.Linux.static.tgz        # [linux]
+  md5: d778d2e6963a5fdbf4edb1f2dafa9f63                                                                       # [linux] 
+  fn: shapeit.v2.r837.MacOSX.tgz                                                                                   # [osx]
+  url: https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.v2.r837.MacOSX.tgz                         # [osx]
+  md5: e356a35bc7a627e670da6dc9cab961d3                                                                       # [osx]
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - boost
```

Comment:
can you add llvm and gcc as we recommend in https://bioconda.github.io/guidelines.html#c-c